Core/GDCore/Project/Object.cpp

```diff
@@ -134,7 +134,6 @@ void Object::UnserializeFrom(gd::Project& project,
 
   objectVariables.UnserializeFrom(
       element.GetChild("variables", 0, "Variables"));
-  behaviors.clear();
 

```

Comment:
Anybreason for removing this clear? Isn't this risky?

Reply:
It would clear default behaviors that are added by the Object constructor before unserialization.

Reply:
Can you do a helper function/lambda (in this file, or at least private) to clear the behaviors that are not default behaviors?

Otherwise we break the contract that calling multiple times UnserializeFrom(...) should keep the "same result" (i.e: in this case, we risk adding more and more duplicated behaviors if we call it more than once)